sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/CollationBenchmark.scala

```diff
@@ -185,6 +185,48 @@ abstract class CollationBenchmarkBase extends BenchmarkBase {
     }
     benchmark.run(relativeTime = true)
   }
+
+  def benchmarkInitCap(
+      collationTypes: Seq[String],
+      utf8Strings: Seq[UTF8String]): Unit = {
+    type collationType = Int
+    type InitCapEstimator = (UTF8String, collationType) => Unit
+    def skipCollationTypeFilter: Any => Boolean = _ => true
+    def createBenchmark(
+        implName: String,
+        impl: InitCapEstimator,
+        collationTypeFilter: String => Boolean): Unit = {
+      val benchmark = new Benchmark(
+        s"collation unit benchmarks - initCap using impl ${implName}",
```

Comment:
nit: the enclosing braces are redundant:
```suggestion
        s"collation unit benchmarks - initCap using $implName",
```

Reply:
Fixed